modules/tests/shared/src/test/scala/lucuma/core/conditions/ConditionsLikelihoodSuite.scala

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.conditions
+
+import coulomb.syntax.*
+import coulomb.units.accepted.*
+import eu.timepit.refined.refineV
+import lucuma.core.enums.CloudExtinction
+import lucuma.core.enums.Site
+import lucuma.core.enums.SkyBackground
+import lucuma.core.enums.WaterVapor
+import lucuma.core.math.Declination
+import lucuma.core.math.Wavelength
+
+/**
+  * Tests the likelihood calculations
+  */
+class ConditionsLikelihoodSuite extends munit.FunSuite:
+
+  test("minimum airmass"):
+    assertEqualsDouble(minimumAirmass(Declination.fromDoubleDegrees(20).get, Site.GN).value.toDouble, 1.000003, 0.000001)
+    assertEqualsDouble(minimumAirmass(Declination.fromDoubleDegrees(80).get, Site.GN).value.toDouble, 2.003746, 0.000001)
+    assertEqualsDouble(minimumAirmass(Declination.fromDoubleDegrees(-30).get, Site.GS).value.toDouble, 1.000006, 0.000001)
+    assertEqualsDouble(minimumAirmass(Declination.fromDoubleDegrees(-90).get, Site.GS).value.toDouble, 1.978889, 0.000001)
+
+  test("percentile sky background"):
+    assertEquals(percentileSkyBackground(SkyBackground.Bright).toPercent, 100.0)
+    assertEquals(percentileSkyBackground(SkyBackground.Dark).toPercent, 50.0)
```

Comment:
Maybe there should be some `Int` syntax to create `CentiPercent`s?  I was wondering why these weren't centi-percents but guess it's probably because they are more awkward to create.

Reply:
Right, this matches user expectation I think, and the original python code